settings.js

```diff
@@ -141,7 +141,7 @@ function setMinimumAssembler(min) {
 function renderFurnace(settings) {
     var furnace = spec.furnace
     if ("furnace" in settings) {
-        furnace = settings.furnace
+        furnace = {name: settings.furnace}
```

Comment:
I don't think this is the right fix. Only the name is used. The variable should be renamed furnaceName, and the name used consistently throughout.